aeron-driver/src/main/c/concurrent/aeron_atomic64_gcc_x86_64.h

```diff
@@ -64,16 +64,6 @@ do \
 } \
 while (false)
 
-#define AERON_CMPXCHG64(original, dst, expected, desired) \
```

Comment:
Why remove this?

Reply:
- It's not used anywhere, `aeron_cmpxchg64` is used instead
 - The MSVC definition used GCC syntax so wouldn't compile if used